Tests/PublisherTests/WorkerQueue/PublisherWorkerFactoryTests.swift

```diff
@@ -0,0 +1,34 @@
+import XCTest
+import AblyAssetTrackingInternal
+import AblyAssetTrackingInternalTesting
+@testable import AblyAssetTrackingPublisher
+import AblyAssetTrackingPublisherTesting
+
+class PublisherWorkerFactoryTests: XCTestCase
+{
+    private let logHandler = InternalLogHandlerMock.configured
+    private var legacyWorkerCalled = false
```

Comment:
Can't this be a local variable in the test function?

Reply:
8a90707